python/ql/test/experimental/dataflow/typetracking-summaries/summaries.py

```diff
@@ -6,10 +6,8 @@
 tainted  # $ tracked
 
 # Lambda summary
-# I think the missing result is expected because type tracking
-# is not allowed to flow back out of a call.
 tainted_lambda = TTS_apply_lambda(lambda x: x, tracked)  # $ tracked
-tainted_lambda  # $ MISSING: tracked
+tainted_lambda  # $ tracked
```

Comment:
It's not entirely clear to me why this is now working (though I'm not one to look a gift horse in the mouth). 🤔